starter-code/clue.js

```diff
@@ -1,92 +1,125 @@
+  occupation: "Scientist"
+};
+
+var profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "Billionare video game designer",
+  age: 22,
+  image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+  occupation: "Designer"
+}
+
+  var missScarlet = {
+  firstName: "Kasandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She is an A-list movie star with a dark past,",
   age: 31,
-  image: https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg,
-  occupation: Actor
-
- Mrs Peacock
-  first name: Eleanor
-  last name: Peacock
-  color: blue
-  description: She is from a wealthy family and uses her status and money to earn popularity
-  age: 36
-  image: https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-  occupation: Socialité
-
- Mr Mustard
-  first name: Jack
-  last name: Mustard
-  color: yellow
-  description: He is a former football player who tries to get by on his former glory
-  age: 62
-  image: https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-  occupation: Retired Football player
-
----- Weapons ----
-
-Rope - Weight: 10
-Knife - Weight: 8
-Candlestick - Weight: 2
-Dumbbell - Weight: 30
-Poison - Weight: 2
-Axe - Weight: 15
-Bat - Weight: 13
-Trophy - Weight: 25
-Pistol - Weight: 20
-
-
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
-
-*/
+  image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation: "Actor"
+  }
+
+ var mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité"
+ }
+
+ var mrMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+ }
+
+ var characters = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+
+
+
+
+var weapons = [
+
+{type: "Rope", weight: 10},
+{type: "Knife", weight: 8},
+{type: "Candlestick", weight:2},
+{type: "Dumbbell", weight: 30},
+{type: "Poison", weight: 2},
+{type: "Axe", weight: 15},
+{type: "Bat", weight: 13},
+{type: "Trophy", weight: 25},
+{type: "Pistol", weight: 20}
+];
+
+var rooms = [
+  "Dinning Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+]
+
+
+function random_selector(array) {
+  return Math.floor(Math.random()*array.length);
```

Comment:
This function is not calle "random_number" but "random selector". It would be expected to return an element of the array, not an index. You can do that this way:
```
function random_selector(array) {
  var index = Math.floor(Math.random()*array.length);
  return array[index];
}
```
I put it in two lines because is easier to read.